apps/ima/fields/index.js

```diff
@@ -2,9 +2,37 @@
 
 module.exports = {
   'who-are-you': {
-    isPageHeading: true,
     mixin: 'radio-group',
     options: ['person-named', 'has-legal-representative', 'someone-else'],
+    validate: 'required',
+    legend: {
+      className: 'visuallyhidden'
+    }
+  },
+  image: {
+    mixin: 'input-file',
+    labelClassName: 'visuallyhidden'
+  },
+  'declaration-immigration-adviser': {
+    isPageHeading: false,
+    mixin: 'radio-group',
+    labelClassName: 'visuallyhidden',
+    options: [{
+      value: 'yes'
+    }, {
+      value: 'no'
+    }],
+    validate: 'required'
+  },
```

Comment:
If this checkbox is left as blank, user can progress with the workflow. Is that correct? Same applies for the other declaration checkboxes.

Reply:
Validation with messages created for checkbox